src/test/CurrencyTest.sol

```diff
@@ -5,23 +5,23 @@ import {Currency, CurrencyLibrary} from "../types/Currency.sol";
 
 contract CurrencyTest {
```

Comment:
why does using the library work here? There no using CurrencyLibrary for Currency statement?

Reply:
Because in the Currency type i wrote `using CurrencyLibrary for Currency global` so now we dont need that line anywhere

Reply:
missed that - thanks!